test/libsolidity/semanticTests/error/selector.sol

```diff
@@ -1,18 +0,0 @@
-library L {
-    error E();
-}
-library S {
-    error E(uint);
-}
-library T {
-    error E();
-}
-contract C {
-    function f() public pure returns (bytes4, bytes4) {
-        assert(L.E.selector == T.E.selector);
-        assert(L.E.selector != S.E.selector);
-        return (L.E.selector, S.E.selector);
-    }
-}
-// ----
-// f() -> 0x92bbf6e800000000000000000000000000000000000000000000000000000000, 0x2ff06700000000000000000000000000000000000000000000000000000000
```

Comment:
This test is completely redundant. Probably should have been deleted when `error_selector.sol` was added.

Reply:
Also, it was in `semanticTests/error/` while we also have `semanticTests/errors/`. I'm merging these two dirs here and leaving only `errors/`.